server/plugins/redis.plugin.js

```diff
@@ -2,19 +2,32 @@
 
 const config = require('../utils/config')
 
+const _getSettings = () => {
+  if (config.redisUseTls) {
+    return {
```

Comment:
There's quite a bit of repeated code here. It would be better to create an object with the host and port, since this is needed in all cases, then optionally add the password and tls properties to the object if needed:

const settings = {
      host: config.redisHost,
      port: config.redisPort,
}

if (config.redisUseTls) {
    settings.tls = {}
}

if (config.redisPassword) {
    password = config.redisPassword
}

return settings

Reply:
Thanks, I've made those changes